meta/saisanitycheck.c

```diff
@@ -435,6 +435,11 @@ bool sai_metadata_is_acl_field_or_action(
         {
             return true;
         }
+
+        if (metadata->isextensionattr == true)
+        {
+            return true;
```

Comment:
problem with this is that, this function checkes whether attr id is acl field or acl action, but if attribute is some other attribute and is extension, then this function will also return true :( and i think there is no good way to annotate that

Reply:
also change "if (metadata->isextensionattr == true)" to "if (metadata->isextensionattr)"

Reply:
Done